README.md

```diff
@@ -23,6 +23,31 @@ This content repository allows easy content management via a headless CMS, using
     - They are built off of the `main` git branch
 
 ## Setting up locally
+
+### Redis
+
+For Linux and WSL2 (Windows Subsystem for Linux)
+ Creating a docker container with the ports exposed, called `cr_redis`
+`docker run -d -p 6379:6379 --name cr_redis redis`
+
+To then run the docker container,
+`docker run cr_redis`
+
+This can work for mac and (possibly Windows) by setting the environment variable `DATABASE_URL=postgres://postgres@0.0.0.0/contentrepo`
```

Comment:
I'm not sure if it's `0.0.0.0`, or maybe `host.docker.internal`? Maybe we leave this out until we can confirm?

Reply:
@rudigiesler the `0.0.0.0` is confirmed to work on mac, @codiebeulaine can you maybe check if `host.docker.internal` works?